pkg/controller/storagecluster/volumesnapshotterclasses.go

```diff
@@ -28,6 +28,12 @@ const (
 	snapshotterSecretNamespace = "csi.storage.k8s.io/snapshotter-secret-namespace"
 )
 
+type SnapshotClassConfiguration struct{
```

Comment:
Exported type requires a comment or the linter fails

Reply:
```suggestion
// SnapshotClassConfiguration provides configuration options for a Snapshot Class
type SnapshotClassConfiguration struct{
```

Reply:
Yeah, this linter requirement is just *so silly* because it encourages people to add meaningless and redundant comments ... :roll_eyes:

Reply:
Some comments may end up being redundant because the type/function name is sufficiently descriptive, but I find that having the linter error forces a change that may guide my brain to examine whether I really understand the struct or if the name could be improved.